src/pages/manage/productions/index.page.js

```diff
@@ -53,23 +53,25 @@ const Index = () => {
   const getProductionsQuery = useGetProductions({
     name: searchInput,
     paginationOptions: {
-      start: currentPageProductions - 1,
+      start: (currentPageProductions - 1) * productionsPerPage,
       limit: productionsPerPage,
     },
   });
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const rawProductions = getProductionsQuery.data?.member ?? [];
 
+  useEffect(() => {
+    if (rawProductions.length > 0) {
+      setActiveProductionId(rawProductions[0].production_id);
+    }
+  }, [rawProductions]);
+
```

Comment:
You can combine the 2 useEffects like this, if I'm not mistaken

```suggestion
  useEffect(() => {
    if (rawProductions.length === 0) {
      setActiveProductionId('');
    } else {
      setActiveProductionId(rawProductions[0].production_id);
    }
  }, [rawProductions]);

```